packages/components/src/tools-panel/tools-panel-header/component.tsx

```diff
@@ -39,7 +39,7 @@ const DefaultControlsGroup = ( {
 	const resetSuffix = <ResetLabel aria-hidden>{ __( 'Reset' ) }</ResetLabel>;
 
 	return (
-		<MenuGroup label={ __( 'Defaults' ) }>
+		<>
```

Comment:
Is this fragment necessary?

Reply:
The fragment is allowing the `DefaultControlsGroup` and `OptionalControlsGroup` to still be used as a JSX component.

Given the merging of these control groups. I'd like to look at refactoring this in a follow-up. It's possible along with these control group components, there could be further clean up needed for the ToolsPanel components.

What do you think?

Reply:
A follow-up sounds great.